research/src/pages/popup/popup.research.explainer.mdx

```diff
@@ -185,107 +344,85 @@ The rules the UA uses to manage these interactions depends on the element types,
-
-
-
-* Applicable to any element.
-* When an element with `delegatesfocus` is focused, the first focusable descendent of that element is focused instead. This is regardless of the element's top-layer status. (This can be added using the [focus delegate spec](https://html.spec.whatwg.org/#focus-delegate).)
-
-The use cases for these attributes need to be better enumerated.
-
-
-## Anchoring (the `anchor` Attribute)
-
-A new attribute, `anchor`, should be added for all elements, whose value is an idref of an “anchoring” element. This anchor relationship is used for two behaviors:
 
 
+## Accessibility / Semantics
 
-1. Establish the provided anchor element as an “ancestor” of this popup, for the purposes of light-dismiss behavior. In other words, when a popup is shown, typically all other popups are closed. An exception would be made for all popups in the ancestor chain formed by the anchor element.
-2. The referenced anchor element could be used by the [Anchor Positioning feature](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/CSSAnchoredPositioning/explainer.md).
+Since the `popup` content attribute can be applied to any element, and this only impacts the element's presentation (top layer vs not top layer), this addition does not have any direct semantic or accessibility impact. The element with the `popup` attribute will keep its existing semantics and AOM representation. For example, `<img popup=popup>` will continue to have `role=img`, but will be able to be displayed on top of other content. Similarly, ARIA can be used to modify accessibility mappings in [the normal way](https://w3c.github.io/html-aria/), for example `<img popup=popup role=presentation>`.
 
+As mentioned in the [Declarative Triggers](#declarative-triggers) section, accessibility mappings will be automatically configured to associate the popup with its trigger element, as needed.
 
-## Accessibility / Semantics
+## Disallowed elements
 
-Since the `popup` content attribute can be applied to any element, and only impacts the element's presentation (top layer vs not top layer), this should not have **any semantic or accessibility impact**. I.e., the element with the `popup` attribute will keep its existing semantics and AOM representation. In the event an author needs to extend or modify a particular element's ARIA semantics, this may be done in accordance to existing allowances of [ARIA in HTML](https://w3c.github.io/html-aria/).
+While the popup API can be used on most elements, there are some limitations. For example, calling `showPopup()` on a `<dialog>` element will result in an exception being thrown, as will calling it on an [active fullscreen element](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen). All other element types are valid.
```

Comment:
I do not think a dialog should be completely barred from being a popup (unless i'm misunderstanding this line?)  it's also in opposition with the example that follows in the next section, re: `<my-date-picker role=dialog id=datepicker popup=popup>`

can expand on this if need be... but maybe this should just be clarified to mean a 'modal' dialog would throw an exception?  (even though there are tons of instances of custom modal dialogs where people implement a click-outside-to-close-light-dismiss... but, i have too many other hills i've decided to die on...)

Reply:
I've gone back and forth on this. But I do see your point here that `role=dialog` semantics might be entirely appropriate on something that has popup behaviors (like click-outside light dismiss).

I've added explicitly "modal" dialog here.